xml/x11_banners.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+X11 vendor fields (part of the X11 success response) are matched 
```

Comment:
Could you provide a little more detail here?  What exactly is the response in response to?  Is this the same vendor that is printed in https://github.com/rapid7/metasploit-framework/blob/master/modules/auxiliary/scanner/x11/open_x11.rb?  If so, eventually someone should improve that to have recog coverage too.